examples/cpp/keyvaluestore/client.cc

```diff
@@ -0,0 +1,85 @@
+/*
+ *
+ * Copyright 2018 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <iostream>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include <grpcpp/grpcpp.h>
+
+#ifdef BAZEL_BUILD
+#include "examples/protos/keyvaluestore.grpc.pb.h"
+#else
+#include "keyvaluestore.grpc.pb.h"
+#endif
+
+using grpc::Channel;
+using grpc::ClientContext;
+using grpc::Status;
+using keyvaluestore::Request;
+using keyvaluestore::Response;
+using keyvaluestore::KeyValueStore;
+
+class KeyValueStoreClient {
+ public:
+  KeyValueStoreClient(std::shared_ptr<Channel> channel)
+      : stub_(KeyValueStore::NewStub(channel)) {}
+
+  // Requests each key in the vector and displays the value as a pair
```

Comment:
"...displays the key and its corresponding value as a pair"

Reply:
done